### PR TITLE
[MOB-4139] Change mobile app event schema

### PR DESCRIPTION
**Pull request by mrmike**
Hi @pchojnacki Could you take a look on this PR?

I've extracted common fields for all events to batch json.

### CODE REVIEW

**Comment by aniaMu** [comment]
ping @pchojnacki

**Comment by pchojnacki** [comment]
service with changes is deployed at:
http://api.wikia-services.com/testing.clickstream/

Since the schema has been changed so much I'll be waiting for green lite
from you when I can deploy it /clickstream/

**Comment by nmonterroso** [comment]
changes LGTM, though tests are failing.

File: service/clickstream/src/main/java/com/wikia/service/clickstream/transform/JsonObjectToFlatEntryStream.java [hunk 1]
```diff
@@ -32,13 +34,17 @@ public JsonObjectToFlatEntryStream() {
     this(DEFAULT_ITEM_SEPARATOR, DEFAULT_ARRAY_FORMAT);
   }
 
-  private Set<Map.Entry<String, String>> recurse(@Nonnull JsonNode node, @Nullable String path) {
+  private Set<Map.Entry<String, String>> recurse(JsonNode node, @Nullable String path) {
```
Comment (drsnyder) [inline service/clickstream/src/main/java/com/wikia/service/clickstream/transform/JsonObjectToFlatEntryStream.java]: Should this be called `flatten`?

**Pull request merged**